typings/vscode-mssql.d.ts

```diff
@@ -247,6 +252,10 @@ declare module 'vscode-mssql' {
         validateStreamingJob(packageFilePath: string, createStreamingJobTsql: string): Thenable<ValidateStreamingJobResult>;
     }
 
+    export interface IAzureFunctionsService {
+        addSqlBinding(bindingType: BindingType, filePath: string, functionName: string, objectName: string, connectionStringSetting: string): Thenable<ResultStatus>;
```

Comment:
Doc comments please. 

And is the connectionStringSetting an actual connection string? If so how does it handle stuff like AAD connections?

Reply:
+1 to docstrings